lib/workers/repository/init/merge.spec.ts

```diff
@@ -174,6 +181,7 @@ describe('workers/repository/init/merge', () => {
 
   describe('mergeRenovateConfig()', () => {
     beforeEach(() => {
+      platform.getRawFile.mockResolvedValueOnce(null);
```

Comment:
```suggestion
```

it's returning `undefined` by default, do you need explicit `null` ?

Reply:
indeed this is not needed